spec/models/event/course_record_spec.rb

```diff
@@ -381,6 +391,88 @@ def new_record(event, attrs = {})
         expect(subject.vollkosten_pro_le).to eq(0.to_d)
       end
     end
+
+    context '#duration_in_days?' do
+      context 'bk' do
+        let(:record) { new_record(event_bk) }
+
+        it 'is expected to be false' do
+          expect(subject.duration_in_days?).to eq(true)
```

Comment:
Interessierte Nachfrage: Könnte man hier nicht auch

```
expect(subject).to be_duration_in_days
```

schreiben? Ich finde es auch so, wie es jetzt ist, nicht falsch und würde es so lassen.

Reply:
Kann man. Das wusste ich gar nicht. Ich werde es noch anpassen, weil ich eh gesehen habe, dass die Beschreibung nicht passt. (Es steht `is expected to be false`, dabei sollte es `true` sein.)